### PR TITLE
Oadp 1181 oadp 1 2 0 release notes hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
GH# : 
OCPBUGS# : [OADP-1181](https://issues.redhat.com//browse/OADP-1181)
OSDOCS# : [OADP-1181](https://issues.redhat.com//browse/OADP-1181)

Version(s): OADP 1.2.0

Issue: https://issues.redhat.com/browse/OADP-1181

Link to docs preview: 

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by HagayVider1** [comment]
@kaovilai 
See new PR for OADP-1181 (https://issues.redhat.com/browse/OADP-1181)

File: modules/oadp-release-notes-1-2-0.adoc [hunk 1]
```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-0_{context}"]
+= OADP 1.2.0 release notes
+
+The OADP 1.2.0 release notes lists any new features, resolve issues and bugs, and known issues.
```
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:9]: resolved issues

**Comment by anarnold97** [comment]
There are still merge conflicts. 

Please rebase and see if this resolves the problem, I suspect it might

**Comment by HagayVider1** [comment]
@anarnold97  @weshayutin 
See updated OADP 1.2.0 release notes.

**Comment by anarnold97** [comment]
@HagayVider1 - thanks. there are still merge conflicts. Let us look tomorrow. The cause is simple, you have not rebased.

**Pull request closed without merging**